bndtools.core/resources/enroute/_plugin.xml

```diff
@@ -1,13 +1,5 @@
 <!-- ENROUTE TEMPLATES -->
 
-<extension point="bndtools.core.cnfTemplates">
-	<template id="enRoute" name="OSGi enRoute Configuration (alpha)"
-		doc="enroute/cnf.xml" paths="enroute/osgi.enroute.template/cnf"
-		priority="0" managed="ext/.*" ignores="/cache/" version="10" category="AAA/Core"
-		nobuild="true" icon="enroute/enroute-bw-16x16.png" />
-</extension>
-
-
```

Comment:
Thats correct, I simply didn't have my master completely up2date.  I'll
resend this without the 2nd commit.

On Wed, Apr 1, 2015 at 11:44 PM, BJ Hargrave <notifications@github.com>
wrote:

> In bndtools.core/resources/enroute/_plugin.xml
> <https://github.com/bndtools/bndtools/pull/1086#discussion_r27581756>:
>
> > @@ -1,13 +1,5 @@
> >  <!-- ENROUTE TEMPLATES -->
> >
> > -<extension point="bndtools.core.cnfTemplates">
> > -	<template id="enRoute" name="OSGi enRoute Configuration (alpha)"
> > -		doc="enroute/cnf.xml" paths="enroute/osgi.enroute.template/cnf"
> > -		priority="0" managed="ext/.*" ignores="/cache/" version="10" category="AAA/Core"
> > -		nobuild="true" icon="enroute/enroute-bw-16x16.png" />
> > -</extension>
> > -
> > -
>
> This seems to be the same as pkriens@c68e502
> <https://github.com/pkriens/bndtools/commit/c68e502fa2d13d6d744323229a276afe08a5e8ff>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bndtools/bndtools/pull/1086/files#r27581756>.
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com